backend/src/main/java/ai/verta/modeldb/versioning/CommitDAORdbImpl.java

```diff
@@ -1192,6 +1192,8 @@ public CommitPaginationDTO findCommits(
           sortKey = ModelDBConstants.DATE_CREATED;
         } else if (sortKey.equals(ModelDBConstants.TIME_UPDATED)) {
           sortKey = ModelDBConstants.DATE_UPDATED;
+        } else if (sortKey.contains(ModelDBConstants.OBSERVATIONS)) {
```

Comment:
I do not understand how this is fixing the tests, please update the PR description